lightning: refined description about duplicate-resolution (#13904)
pingcap/docs: tidb-lightning/tidb-lightning-configuration.md

# Whether to detect and resolve duplicate records (unique key conflict) in the physical import mode.
# The following resolution algorithms are supported:
#  - none: does not detect duplicate records, which has the best performance of the three algorithms.
#    But if there are duplicate records in the data source, it might lead to inconsistent data in the target TiDB.

> #  - remove: If there are primary key or unique key conflicts between the inserting data A and the existing data B in the target table, the conflicting data A and B will be removed from the target table and recorded in the lightning_task_info.conflict_error_v1 table in the target TiDB.   Users can manually reinsert the correct records into the target table based on their business requirements. 
> #    Note that the required version of the target TiKV is no earlier than v5.2.0;
> #    otherwise it falls back to 'none'.

Reviewer: ```suggestion
#  - remove: If there are primary key or unique key conflicts between the inserting data A and the existing data B in the target table, A and B will be removed from the target table and recorded in the `lightning_task_info.conflict_error_v1` table in the target TiDB. You can manually insert the correct records into the target table based on your business requirements. 
# Note that the target TiKV must be v5.2.0 or later versions; otherwise it falls back to 'none'.
```

# Whether to detect and resolve duplicate records (unique key conflict) in the physical import mode.
# The following resolution algorithms are supported:
#  - none: does not detect duplicate records, which has the best performance of the three algorithms.
#          But if there are duplicate records in the data source, it might lead to inconsistent data in the target TiDB.
#  - remove: if there are primary key or unique key conflicts between the inserting data A and the existing data B 
#            in the target table, A and B will be removed from the target table and recorded
#            in the `lightning_task_info.conflict_error_v1` table in the target TiDB. 
#            You can manually insert the correct records into the target table based on your business requirements. 

> #            Note that the target TiKV must be v5.2.0 or later versions; otherwise it falls back to 'none'.

Reviewer: ```suggestion
#            Note that the target TiKV must be v5.2.0 or later versions; otherwise it falls back to 'none'.
# The default value is 'none'.
```

# Whether to detect and resolve duplicate records (unique key conflict) in the physical import mode.
# The following resolution algorithms are supported:

> #  - none: does not detect duplicate records, which has the best performance of the three algorithms.

Reviewer 2: ```suggestion
#  - none: does not detect duplicate records, which has the best performance of the two algorithms.
```

#          But if there are duplicate records in the data source, it might lead to inconsistent data in the target TiDB.

> #  - remove: if there are primary key or unique key conflicts between the inserting data A and the existing data B 

Reviewer 3: ```suggestion
#  - remove: if there are primary key or unique key conflicts between the inserting data A and B 
```

> #  - remove: if there are primary key or unique key conflicts between the inserting data A and the existing data B 
> #            in the target table, A and B will be removed from the target table and recorded

Reviewer: ```suggestion
#  - remove: if there are primary key or unique key conflicts between the inserting data A and B, 
#            A and B will be removed from the target table and recorded
```
